### PR TITLE
WP-4031: Refactor forEach -> for loops

### PR DESCRIPTION
**Pull request by mikeerickson**
Please see [Webpack issue #4031](https://github.com/webpack/webpack/issues/4031) for details

### CODE REVIEW

**Comment by TheLarkInn** [comment]
Thanks a bunch can you run `npm run beautify` to fix the indentation and linting.

File: package.json [hunk 1]
```diff
@@ -39,7 +39,9 @@
     "beautify": "beautify-rewrite lib/**.js test/*.js",
     "lint": "eslint lib test",
     "pretest": "npm run lint && npm run beautify-lint",
-    "test": "mocha --full-trace --check-leaks",
+    "test_old": "mocha --full-trace --check-leaks",
```
Comment (TheLarkInn) [inline package.json]: Is this meant to be here? Is this meant to be here or were you thinking about reverting back.

File: package.json [hunk 2]
```diff
@@ -39,7 +39,9 @@
...
+    "test": "mocha --check-leaks",
+    "test:watch": "mocha --check-leaks",
```
Comment (TheLarkInn) [inline package.json]: Maybe dupe that should be reviewed.

**Comment by mikesherov** [comment]
Thanks for contributing, @mikeerickson. As this pull request has gone stale since changes being requested, I'm going to close this PR. 

If you feel like picking this back up again, feel free to open a new one with the requested changes!

**Pull request closed without merging**